app/assets/stylesheets/public/_toc.scss

```diff
@@ -1,44 +1,108 @@
 .Toc {
+  background-color: #f8f8f8;
```

Comment:
Is there a variable kicking around for this one? Feels like we should be pulling a few of these settings from a single point to minimise the blast radius of potential future style changes.

Reply:
Agree 👍 Will start adding variables to Docs.

I've exported the color variables from our Figma Library and putted them in `_variables.scss` and assigned them in https://github.com/buildkite/docs/pull/1851/commits/e58164db0ebbea65efabe5255883bb1c0a9aed20

I've done this for `_toc.scss`, replaced all instances of the lime color, and will gradually roll this out for other hex colors in a future card to keep this PR small for now.